app/src/main/java/com/sdpteam/connectout/event/creator/LocationPicker.java

```diff
@@ -12,26 +12,28 @@
 import com.google.android.gms.maps.model.Marker;
 import com.google.android.gms.maps.model.MarkerOptions;
 import com.sdpteam.connectout.event.Event;
-import com.sdpteam.connectout.event.EventsViewModel;
+import com.sdpteam.connectout.event.nearbyEvents.EventsViewModel;
+import com.sdpteam.connectout.event.nearbyEvents.map.EventsMapViewFragment;
 
 import androidx.annotation.NonNull;
 
-public class PositionSelectorFragment extends MapViewFragment implements OnMapReadyCallback {
+// TODO : it is very strange to make a fragment extend another one. especially when they have different purposes.
```

Comment:
Let's open an issue to refactor this independently of this PR

Reply:
https://github.com/ConnectOut-sdp/sdp2023/issues/58